adserver/analyzer/tests.py

```diff
@@ -174,8 +174,8 @@ def test_analyzer_success(self):
 
                 <!-- Only this should be used -->
                 <main>
-                <p>Data science and machine learning are important</p>
```

Comment:
Something changed with the lemmatizer and using the built-in model. It is lemmatizing differently than before. This does *not* affect our trained model but only the built-in model.